### ISSUE TITLE
Not avaiable for Ubuntu Focal via apt-get

### ISSUE THREAD

**Issue by JeffResc** [zzag/plasma5-wallpapers-dynamic#67]
The readme states that this is just a simple apt-get away, yet my system is unable to find it. Am I missing something?

```
jeff@G7-7588:~$ sudo apt-get update
Hit:1 http://us.archive.ubuntu.com/ubuntu focal InRelease
Get:2 http://us.archive.ubuntu.com/ubuntu focal-updates InRelease [111 kB]                   
Hit:3 http://packages.microsoft.com/repos/vscode stable InRelease                                                                           
Get:4 http://us.archive.ubuntu.com/ubuntu focal-backports InRelease [98.3 kB]                                                               
Get:5 http://security.ubuntu.com/ubuntu focal-security InRelease [107 kB]
Get:6 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [232 kB]
Get:7 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [205 kB]
Get:8 http://us.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 DEP-11 Metadata [2,468 B]
Get:9 http://us.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [1,768 B]
Get:10 http://security.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [24.2 kB]      
Get:11 http://security.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [56.5 kB]
Fetched 838 kB in 1s (938 kB/s)             
Reading package lists... Done
jeff@G7-7588:~$ sudo apt-get install plasma-wallpaper-dynamic
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package plasma-wallpaper-dynamic
```

**Comment by zzag** [comment]
plasma-wallpaper-dynamic package is available in ubuntu 20.10, but it seems like you're running 20.04.

**Comment by JeffResc** [comment]
Yes, Ubuntu Focal is 20.04. Will it be available for 20.04 or only 20.10?

**Comment by zzag** [comment]
As far as I know, it's available only in 20.10.
